civihr_employee_portal/src/Forms/OnboardingWebForm.php

```diff
@@ -190,4 +191,30 @@ private function userCreatedAfterOnboardingReleased() {
     return $user->created > $onboardingRelease;
   }
 
+  /**
+   * Unfortunately for us the contact profile page will expect an image URL
+   * using the civicrm/file?photo=foo.jpg style. Since our contact images are
+   * created using webform they won't match this so to avoid the warnings about
+   * 'Undefined index: photo' we append photo=0 here. I need a shower.
```

Comment:
Please leave a line before @see comment.
You can remove the I need a shower comment too  :)